core/src/data/relation.rs

```diff
@@ -51,6 +55,5 @@ impl<'a> Table<'a> {
 
```

Comment:
I think we can reduce or remove `Relation` struct dependency.

For a small single step,
we can consider to add `get_alias` which directly takes `TableFactor`.
Then, no more making `Relation` to get_alias
```rust
pub fn get_alias(table_factor: &TableFactor) -> Result<&String> {
    match table_factor {
        TableFactor::Table { 
            name, alias: None, ..
        } => get_name(name),
        TableFactor::Table {
            alias: Some(TableAlias { name, .. }),
            ..
        }
        | TableFactor::Derived {
            alias: TableAlias { name, .. },
            ..
        } => Ok(name),
    }   
}   
```

Reply:
I removed Relation and implemented get_name, get_alias, get_index at fetch.rs
Also realized that there were two `get_name` so that i renamed `get_name(&ObjectName)` to `fetch_name(&Object_name)`